packages/openneuro-app/src/scripts/refactor_2021/search/use-search-results.tsx

```diff
@@ -148,7 +149,14 @@ export const useSearchResults = () => {
 
   const boolQuery = new BoolQuery()
   if (keywords.length)
-    boolQuery.addClause('must', simpleQueryString(sqsJoinWithAND(keywords)))
+    boolQuery.addClause(
+      'must',
+      simpleQueryString(sqsJoinWithAND(keywords), [
+        'latestSnapshot.readme',
+        'latestSnapshot.description.Name',
+        'latestSnapshot.description.Authors',
+      ]),
+    )
```

Comment:
This is working for me, tested datasets with mutually exclusive data in each field and got correct matches in the query results.

Reply:
so authors for me will show the dataset for “First Last” but not for “First” or “Last”
Also, if I search for multiple authors I do not get multiple results as the help text suggests.

> Results on multiple inputs will include all datasets that have ANY of the entered names

As for the keyword search.  When I search for keywords in the title, it returns results but the keyword is not in the title, it is in the description (or some fuzzy match is).

Sometimes when I search the keyword for a specific term in the title I get 0 results.

Reply:
@nellh if this is working for you and you think it might be local issues and/or GNO issues I am good to merge and see it on stg. lmk